src/query/ts/m3db/encoded_series_iterator.go

```diff
@@ -22,7 +22,6 @@ package m3db
 
```

Comment:
Maybe not in this PR, but we should organize all of these iterator files a bit better.

Reply:
Yeah that's fair, it's kind of in a weird path right now